components/footer.js

```diff
@@ -85,11 +86,12 @@ const Footer = ({ intl }) => (
           <FooterLink href='https://openobservatory.slack.com/' label={intl.formatMessage({ id: 'Footer.Link.Slack' })} />
         </FooterBox>
       </Flex>
-      <Flex flexWrap='wrap'>
+      <Flex flexWrap='wrap' justifyContent='space-between'>
```

Comment:
You can drop this change. It is useful only when there are multiple child nodes inside `<Flex>`. This is my [go-to link](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for all things flex.

Reply:
I forgot to remove it after experimenting with the position of the Version. That's all. Not like I left it on purpose.